<launch>
  <node name="staubli_controller" pkg="staubliTX60"  type="staubliTX60_server" args="http://192.168.11.254:5653/" respawn="true" />
</launch>